src/nvim/options.lua

```diff
@@ -2002,6 +2002,13 @@ local options = {
         		character, the cursor won't move. When not included,
         		the cursor would skip over it and jump to the
         		following occurrence.
+                                                                *cpo-~*
+                ~       When included, don't resolve symbolic links when
+                        changing directory with |:cd|, |:lcd|, or |:tcd|.
+                        This preserves the symbolic link path in buffer names
+                        and when displaying the current directory.  When
+                        excluded (default), symbolic links are resolved to
+                        their target paths.
```

Comment:
Please following the indent of other items
```suggestion
        							*cpo-~*
        	~	When included, don't resolve symbolic links when
        		changing directory with |:cd|, |:lcd|, or |:tcd|.
        		This preserves the symbolic link path in buffer names
        		and when displaying the current directory.  When
        		excluded (default), symbolic links are resolved to
        		their target paths.
```

Reply:
oh it's tab.. i set the tab symbol to space in `listchars` ...then i got mixed up sry.....